src/components/StripeBankAccountTokenInputField/StripeBankAccountTokenInputField.js

```diff
@@ -150,19 +149,16 @@ class TokenInputFieldComponent extends Component {
         return result.token.id;
       })
       .then(token => {
+        // Check if value has changed during async call.
         const changedValues = inputsNeeded.filter(
-          inputType => values[inputType] !== cleanedString(this.state[inputType].value)
+          inputType => values[inputType] !== this.state[inputType].value
         );
         const valuesAreUnchanged = changedValues.length === 0;
 
         // Handle response only if the input values haven't changed
         if (this._isMounted && valuesAreUnchanged) {
           this.setState(prevState => {
-            const errorsClearedFromInputs = inputsNeeded.map(inputType => {
-              const input = prevState[inputType];
-              return { ...input, error: null };
-            });
```

Comment:
This code actually creates another set of input values to state - it's about array vs object: `state[0]` instead of overwriting `state[inputType]`. Reducer would easily solve this, but in the end, this is dead code, since error clearing is done earlier.